tests/ga/test_monitor.py

```diff
@@ -139,184 +134,146 @@ def test_parse_json_event(self, *args):
         self.assertNotEqual(0, event.parameters)
         self.assertTrue(all(param is not None for param in event.parameters))
 
-    def test_add_sysinfo_should_honor_sysinfo_values_from_agent_for_agent_events(self, *args):
-        data_str = load_data('ext/event_from_agent.json')
-        event = parse_json_event(data_str)
-
-        monitor_handler = get_monitor_handler()
-
-        sysinfo_vm_name_value = "sysinfo_dummy_vm"
-        sysinfo_tenant_name_value = "sysinfo_dummy_tenant"
-        sysinfo_role_name_value = "sysinfo_dummy_role"
-        sysinfo_role_instance_name_value = "sysinfo_dummy_role_instance"
-        sysinfo_execution_mode_value = "sysinfo_IAAS"
-        container_id_value = "TEST-CONTAINER-ID-ALREADY-PRESENT-GUID"
-        GAVersion_value = "WALinuxAgent-2.2.44"
-        OpcodeName_value = "2019-11-02 01:42:49.188030"
-        EventTid_value = 140240384030528
-        EventPid_value = 108573
-        TaskName_value = "ExtHandler"
-        KeywordName_value = ""
-
+    @staticmethod
+    def create_mock_sysinfo():
+        os_version_param = "OSVersion"
+        execution_mode_param = "ExecutionMode"
+        ram_param = "RAM"
+        processors_param = "Processors"
         vm_name_param = "VMName"
         tenant_name_param = "TenantName"
         role_name_param = "RoleName"
         role_instance_name_param = "RoleInstanceName"
-        execution_mode_param = "ExecutionMode"
-        container_id_param = "ContainerId"
-        GAVersion_param = "GAVersion"
-        OpcodeName_param = "OpcodeName"
-        EventTid_param = "EventTid"
-        EventPid_param = "EventPid"
-        TaskName_param = "TaskName"
-        KeywordName_param = "KeywordName"
+        location_param = "Location"
```

Comment:
`Location` is populated in `self.sysinfo` in the `MonitorHandler`, but it is not a part of the database schema, so it would always be dropped in the new telemetry pipeline.